http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/api/AppV2Routes.scala

```diff
@@ -177,6 +177,8 @@ object AppV2Routes {
         ea <- x.downField("extraArgs").as[Option[List[String]]]
         wsi <- x.downField("workspaceId").as[Option[WorkspaceId]]
         swi <- x.downField("sourceWorkspaceId").as[Option[WorkspaceId]]
+        adtm <- x.downField("autodeleteThresholdMin").as[Option[Int]]
+        adte <- x.downField("autodeleteEnabled").as[Option[Boolean]]
```

Comment:
... only putting this here based on the comment above, feel free to choose how you see fit (it's just a variable name after all)
```suggestion
        ae <- x.downField("autodeleteEnabled").as[Option[Boolean]]
```